Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -324,7 +411,7 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
     if ($totalServiceFabricWithoutEncryptandSign  -eq 0)
```

Comment:
line 399 : Make it info message type

Reply:
Write-Host "Separating Service Fabric(s) for which Cluster Protection level is not as Encrypt and Sign or no Cluster Protection level is not enabled..."  this line